src/components/heroSection.tsx

```diff
@@ -9,7 +9,7 @@ const HeroSection: React.FC<HeroSectionProps> = ({ title, subtitle }) => {
   return (
     <div className="flex flex-1 px-28">
       <div className="space-y-9 flex flex-col flex-1">
-        <h1 className="text-6xl">{title}</h1>
+        <h1 className="text-6xl font-medium">{title}</h1>
```

Comment:
🤔 isn't this font medium by default?

Reply:
That's what I thought too, but then inspecting the element I saw a font-weight: inherit; I'm not sure, but I think that tailwind is resetting all the tags styles, that's how the h1 looks like without any style:
<img width="1126" alt="image" src="https://github.com/user-attachments/assets/d2b7f1f8-7c50-4140-b465-65a1dc51e452">

Reply:
you're right in thinking that TW resets all tags styles 😄  https://tailwindcss.com/docs/preflight